src/sammich/plugins/reminder.py

```diff
@@ -0,0 +1,113 @@
+import datetime
+import logging
+import re
+import time
+
+import requests
+from dotenv import dotenv_values
+from machine.plugins.base import MachineBasePlugin
+from machine.plugins.decorators import command
+
+secrets = dotenv_values(".secrets")
+
+SLACK_BOT_TOKEN = secrets["SLACK_BOT_TOKEN"]
+
+
+class ReminderPlugin(MachineBasePlugin):
+    def parse_command(self, command_text):
+        pattern = re.compile(r"\[(.*?)\]\s+\[(.+?)\]\s+\[(.+?)\]")
```

Comment:
I suggest moving this to the module level and giving more descriptive name.

Reply:
Please check if its okay now !, thanks